src/go/rpk/pkg/cli/registry/schema/get.go

```diff
@@ -123,7 +130,19 @@ potential (mutually exclusive) ways:
 	cmd.Flags().StringVar(&schemaFile, "schema", "", "Schema file to check existence of, must be .avro or .proto; subject required")
 	cmd.Flags().StringVar(&schemaType, "type", "", fmt.Sprintf("Schema type of the file used to lookup (%v); overrides schema file extension", strings.Join(supportedTypes, ",")))
 	cmd.Flags().BoolVar(&deleted, "deleted", false, "If true, also return deleted schemas")
+	cmd.Flags().BoolVar(&printSchema, "print-schema", false, "If true, print the schema JSON")
```

Comment:
What do you think about making this a NoOptDefValue="true" (or whatever the name is)?

Try to run strconv.ParseBool on the flag value, if that returns true: print the schema as one long line. If ParseBool returns false, if the input is "pretty", pretty print the schema. Otherwise, error.

Reply:
Sounds good, I'll go with that. However, the proto records are stored with newlines so they will be `pretty` by default.

Reply:
Good point, I forgot about that. Then in that case maybe we should do pretty by default, always? And you can ignore my thought for now?